gammapy/data/gti.py

```diff
@@ -438,6 +438,56 @@ def union(self, overlap_ok=True, merge_equal=True):
         merged = Table(rows=merged, names=["START", "STOP"], meta=self.table.meta)
         return self.__class__(merged, reference_time=self.time_ref)
 
+    def intersect(self, other, padding=0.0):
+        """Intersection of overlapping time intervals between two GTIs.
+
+        Returns a new `~gammapy.data.GTI` object.
+
+        Parameters
+        ----------
+        other : `~gammapy.data.GTI`
+            Another GTI object to intersect with.
+        padding : float, optional
```

Comment:
Why padding is necessary? On GTIs you would not want to change an interval right?